tests/unit/AlterationSummary.spec.ts

```diff
@@ -142,6 +144,6 @@ describe('Alteration Summary component', () => {
     const divs = wrapper.findAll('.alteration-date-time .v-card')
     expect(divs.length).toBe(2)
     expect(divs.at(1).text()).toContain('The alteration for this business will be effective as of:')
-    expect(divs.at(1).text()).toContain('Friday, March 5, 2021 at 8:30 am Pacific time')
+    expect(divs.at(1).text()).toContain('Friday, March 5, 2021 at 8:30 AM Pacific time')
```

Comment:
No, it should be "am" in lower case. Maybe I missed a datetime formatter somewhere. If you find it, can you fix pls?

Reply:
Ref: https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/web-content-development-guides/web-style-guide/writing-guide/numbers#years-and-time

Reply:
K, this is my bad. I shouldn't have changed this to pass the test, will root this out!